File: CPP/prime.cpp
#include<iostream>
using namespace std;

int main()
{
    int number, temp=0;

    cout << "Enter a positive number: ";
    cin >> number;

    for (int i=2; i<number; i++)
    {
        if (number % i == 0)
        {
            temp++;
            break;
        }
    }
    if (temp)
    {
        cout << number << " is not a prime number";
    }
    else if (number == 1)
    {
        cout << number << " is not a prime number";
    }
    else if (number == 2)
    {
        cout << number << " is a prime number";
    }
    else
    {
        cout << number << " is a prime number";
    }
    

    return 0;
}

File: R/prime.R
a = as.numeric(readline("Enter a number: "))

is_prime <- function(a) {
  for(i in 2:(a-1)) {
    if (a%%i == 0) {
      return(FALSE)
    }
  }
  return(TRUE)
}

print(is_prime(a))